docs/java/features/odata/use-vdm.md

```diff
@@ -466,3 +603,36 @@ POST /ODataService/API_BUSINESS_PARTNER/A_BusinessPartner(123)/to_BusinessPartne
 
 </TabItem>
 </Tabs>
+
+## Migrate to improved OData VDM (Beta)
+The SAP Cloud SDK released an experimental new OData VDM implementation. It improves performance of your Java apps by avoiding unnecessary OData metadata calls and thereby saving roundtrips to the backend system. Also, the Cloud SDK team implemented an OData client library that serves both OData V2 and V4 protocols. Consequently, this approach allows for faster delivery of new features in the future. Note that this new implementation is in Beta state. We'll announce the production readiness explicitly in our release channels.
+
+The following steps outline how to migrate your existing code to leverage the improved OData VDM implementation.
```

Comment:
```suggestion
You can easily switch to the new implementation by replacing any call to `execute()` with `executeRequest()`.
For example:
```

Reply:
I would object because the proposed formulation mixes the statement that the steps follow now with already the first step.